index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Leading Design Festival</title>
+</head>
+<body>
+  <header>
+  <button type="button" class="hamburger">
+    <span></span>
+    <span></span>
+    <span></span>
+  </button>
+  <div class="dNav">
+    <a class = "brand" href="index.html"><img src="./Logo/ld.png" alt=""> Leading Design Festival</a>
+    <ul>
+      <li><a href="index.html">Home</a></li>
+      <li><a href="about.html">About</a></li>
+      <li><a href="">Program</a></li>
+      <li><a href="">Join</a></li>
+      <li><a href="">Sponsor</a></li>
+    </ul>
+    <a class="Campaign" href="#">LD Campaign</a>
+  </div>
+  </header>
+  <section class="intro">
+    <span class="hello">"hello Sharing world"</span>
+    <h1>LEADING DESIGN FESTIVAL GLOBAL SUMMIT 2021</h1>
+    <p class="description">Was a month of amazing design
+      leadership activities including a three
+        day conference, weekly talks, 
+        masterclasses, mentoring sessions and a 
+        host of other events.</p>
+    <span class="date">2021.03.15(THU) ~ 16(FRI)</span>
+    <p class="broughtBy">@Clearleft.</p>
+  </section>
+  <div class="bg"></div>
+  <section class="mainProgram">
+  <h2>Main program</h2>
+  <span class="line"></span>
+  <div class="container">
+        <div class="card">
+          <img src="./assest/icons/program01.png" alt="">
+          <span>Lecture</span>
+          <p>Listen to all speakers from various regions and learn about trends in new normal business in all the world.</p>
+        </div>
+        <div class="card">
+          <img src="./assest/icons/program02.png" alt="">
+          <span>CC Exhibition</span>
+          <p>Let's meet the innovators from various fields who share their latest used technologies in new normal business.</p>
+        </div>
+        <div class="card">
+          <img src="./assest/icons/program03.png" alt="">
+          <span>Forum</span>
+          <p>It's the time to share our thoughts and opinions with experts and learn about the new normal business</p>
+      </div>
+      <div class="card">
```

Comment:
- [ ]  The HTML uses semantic tags, which means,: :innocent:  sections use `section`, articles use `article`..., Kindly for each card place them in article tags and not divs. `<article class="card"></article>` 👍🏻 It could be something like this :point_down: guess that make sense 😅 

```
        <article class="card"> 
               <<img src="./assest/icons/program01.png" alt="">
               <span>Lecture</span>
               <p>Listen to all speakers from various regions and learn about trends in new normal .....</p>
        </article>
```
- I know you can do it 😄 💪🏻 

- [ ] Kindly make sure to add the white missing `border` effect when the user hover any program-list of the program section as shown in the image below 👇🏻 :

- [Cindy Shin](https://www.behance.net/adagio07)[Cindy Shin](https://www.behance.net/adagio07) design:

![main](https://user-images.githubusercontent.com/79058364/169443599-e58401a3-358c-4dab-ae4f-b51f3638bbf9.png)

- Your Design:

![30main](https://user-images.githubusercontent.com/79058364/176779053-46c4cbda-6ce2-4431-a672-9bff0ac7a870.png)


- I know you can do it 😃 💪🏻

Reply:
Hello @BenMukebo,
Thanks for your effort your review is amazing it includes every single detail…
But to this one
Isn’t this a personal preference ?
I mean there is a hover effect,
But dose it have to be the one you pick ?